src/sentry/api/endpoints/team_time_to_resolution.py

```diff
@@ -0,0 +1,58 @@
+from collections import defaultdict
+from datetime import timedelta
+
+from django.db.models import Avg, F
+from django.db.models.functions import TruncDay
+from rest_framework.response import Response
+
+from sentry.api.base import EnvironmentMixin
+from sentry.api.bases.team import TeamEndpoint
+from sentry.api.utils import get_date_range_from_params
+from sentry.models import GroupHistory, GroupHistoryStatus, Project
+
+
+class TeamTimeToResolutionEndpoint(TeamEndpoint, EnvironmentMixin):
+    def get(self, request, team):
+        """
+        Return a a time bucketed list of mean group resolution times for a given team.
+        """
+        project_list = Project.objects.get_for_team_ids(team_ids=[team.id])
+        start, end = get_date_range_from_params(request.GET)
+        end = end.replace(hour=23, minute=59, second=59, microsecond=999999)
+        start = start.replace(hour=0, minute=0, second=0, microsecond=0) + timedelta(days=1)
+        history_list = (
+            GroupHistory.objects.filter(
+                status=GroupHistoryStatus.RESOLVED,
+                project__in=project_list,
+                date_added__gte=start,
+                date_added__lte=end,
+            )
+            .annotate(bucket=TruncDay("date_added"))
+            .values("bucket", "prev_history_date")
+            .annotate(ttr=F("date_added") - F("prev_history_date"))
+            .annotate(avg_ttr=Avg("ttr"))
+        )
+        sums = defaultdict(dict)
```

Comment:
```suggestion
        sums = defaultdict(lambda: {"sum": 0, "count": 0})
```
If you use a lambda to fill out the default values of the dict, then you should be able to skip the if/else and just always increment.